pyspeckit/cubes/SpectralCube.py

```diff
@@ -267,8 +267,29 @@ def copy(self,deep=True):
 
         return newcube
 
+    def _update_header_from_xarr(self):
+        """Uses SpectroscopiAxis' _make_header method to update Cube header"""
+        self.header['NAXIS3'] = self.xarr.size
+
+        self.xarr._make_header()
+        sp_naxis = self._spectral_axis_number
+
+        newhead = {(key.replace('1', str(sp_naxis))
```

Comment:
add a comment here stating "Translates header kws, e.g. CRPIX1->CRPIX3"; it took me a moment to figure that out.